test/fixtures/scenarioManager.js

```diff
@@ -6,9 +6,10 @@ var config = require('./config').scenarios;
 function ScenarioManager(server) {
   if (!server) throw new Error('No server defined');
 
+  // NOTE: some large sets of test data can take several minutes to load
   this.client = new elasticsearch.Client({
     host: server,
-    requestTimeout: 300000
+    requestTimeout: 3000000
```

Comment:
Yep, we might have hit the longer timeout on Travis.  Not sure at this point.  But I can see from current logs it's taking less than 1 minute.  Changing it back to 5 minutes (we may add larger test data at some point).